### ISSUE TITLE
Inspect Element in the Right Click Menu Not Working

### ISSUE THREAD

**Issue by Muketsu** [pulsar-edit/pulsar#516]
### Thanks in advance for your bug report!

- [X] Have you reproduced issue in [safe mode](https://pulsar-edit.dev/docs/atom-archive/faq/sections/what-does-safe-mode-do.html)?
- [X] Have you used the [debugging guide](https://pulsar-edit.dev/docs/atom-archive/hacking-atom/#debugging) to try to resolve the issue?
- [X] Have you checked our [FAQs](https://pulsar-edit.dev/docs/launch-manual/sections/faq/) to make sure your question isn't answered there?
- [X] Have you checked to make sure your issue does not already [exist](https://github.com/issues?q=is%3Aissue+user%3Apulsar-edit+is%3Aopen)?
- [X] Have you checked you are on the latest release of Pulsar?

### What happened?

I apologize if I'm missing something obvious!  I did look through the issues already posted and tried going through the troubleshooting steps, but I'm new to this so I might've made a mistake on my end.

But in dev mode, the right-click inspect element option in the context menu isn't doing anything.  It does show up in the right-click context menu, but there's just no response when it's pressed.  This also seems to be happening only on Mac.  I'm not sure if it's the ARM architecture or the OS, but I wasn't seeing this issue with the same version of Pulsar on an x86-64 machine with Windows 10.  Also, elements are still inspectable using the inspection tool from the development tools panel, the issue seems to be specifically only with the right-click inspect elements option.  I did also try reseting Pulsar to factory settings as mentioned in the debugging guide, so I don't think the issue is related to a third party package.

### Pulsar version

1.104.0 (also tried the rolling release)

### Which OS does this happen on?

🍎 macOS

### OS details

macOS 13.3.1 (a)

### Which CPU architecture are you running this on?

Apple M1/M2

### What steps are needed to reproduce this?

1. In macOS 13.3.1 (a) on an ARM Mac, open the latest version of Pulsar
2. Should happen with any configuration, I tried the factory default settings
3. Go to View -> Open in Dev Mode -> select a file or folder
4. With the file open in Dev Mode, try right-clicking any element in the code, then select "Inspect Element"
5. Nothing should happen

**Comment by confused-Techie** [comment]
Thanks a ton for the report, I'm curious if anyone else from MacOS can replicate this

**Comment by savetheclocktower** [comment]
I can. It's a bit annoying. But the button in the corner of the “Elements” pane works just fine.

**Comment by confused-Techie** [comment]
When you attempt to right click and 'Inspect Element' do you get any logs at all?

**Comment by Muketsu** [comment]
Yeah, that's exactly what I'm seeing too, the corner button is still working, it's just the right click menu.  Definitely not a big deal at all and I'm still using Pulsar as my main editor, but I figured I'd mention it since it's probably not a commonly encountered bug because of how specific it is.

I reproduced the error and checked the console for any error messages, and it doesn't look like anything is being generated when I try using the right click inspect element.

**Comment by S1lverCr0w** [comment]
I'm having the same issue on Linux flatpack and aur packages.

Dev tools aren't working at all and inspect element button not working.

I'll have to get back with an update after trying the  troubleshooting guide and trying it on windows too. 
Might just be my hardware.

**Comment by confused-Techie** [comment]
@S1lverCr0w If DevTools don't open via the keyboard command that's a separate issue, and will likely be resolved by using `--no-sandbox` flag when starting Pulsar from the command line. But if you're issue is just opening from the right click menu then it still does fit in here. Just wanting to make sure we are clear

**Comment by S1lverCr0w** [comment]
Thanks, --no-sandbox has indeed fixed one of my issues where I couldn't in open DevTools via keyboard command.
 
 I still have the issue of opening from the right click menu.
 
 thanks for getting back.


> @S1lverCr0w If DevTools don't open via the keyboard command that's a separate issue, and will likely be resolved by using `--no-sandbox` flag when starting Pulsar from the command line. But if you're issue is just opening from the right click menu then it still does fit in here. Just wanting to make sure we are clear

**Comment by savetheclocktower** [comment]
This is now working for me in 1.131.1 on macOS, so I'm hoping it's fixed for everyone.

If this affected you at any point and you're now running 1.131 or later, please weigh in and let us know if it's still happening.
